packages/data-stores/src/queries/use-has-seen-whats-new-modal-query.ts

```diff
@@ -26,7 +26,7 @@ export const useHasSeenWhatsNewModalQuery = ( siteId: number | null ) => {
 				apiNamespace: 'wpcom/v2',
 			} ),
 		{
-			enabled: !! siteId,
+			enabled: isSimpleSite,
```

Comment:
Why did you remove the `!! siteId` check? Could this be  `!! siteId && isSimpleSite`?